### PR TITLE
New drops aren't created during chain

### PR DESCRIPTION
**Pull request by xai3**

### CODE REVIEW

File: ochimono.rb [hunk 1]
```diff
@@ -100,6 +93,10 @@ def find_connected_drops(origin_drop, connected_drops)
     connected_drops
   end
 
+  def fixed?
+    @fixed_drops.select { |drop| can_fall?(drop) }.count == 0
```
Comment (tyamagu2) [inline ochimono.rb]: `@fixed_drops.none? { |drop| can_fall?(drop) }`

File: ochimono.rb [hunk 2]
```diff
@@ -51,22 +51,15 @@ def can_fall?(drop)
...
   def fall_drops
-    @drops.each do |drop| 
-      if can_fall?(drop)
-        drop.y += Y_VELOCITY
-      else
-        drop.y = drop.y.floor
-        @fixed_drops << drop
-        @drops.delete(drop)
-      end
+    landing = @drops.select { |drop| !can_fall?(drop) }.count != 0
```
Comment (tyamagu2) [inline ochimono.rb]: `@drops.any? { |drop| !can_fall?(drop) }`

**Comment by xai3** [comment]
@tyamagu2 done!!!👮

**Comment by xai3** [comment]
@kota Could you merge this? Thanks.

**Comment by kota** [comment]
@yukiasai Merged. Thanks.

**Pull request merged**